core/src/main/java/org/testcontainers/containers/ContainerState.java

```diff
@@ -260,8 +260,8 @@ default void copyFileToContainer(MountableFile mountableFile, String containerPa
      */
     @SneakyThrows(IOException.class)
     default void copyFileToContainer(Transferable transferable, String containerPath) {
-        if (!isCreated()) {
-            throw new IllegalStateException("copyFileToContainer can only be used with created / running container");
+        if (!InternalMethods.exists(this)) {
```

Comment:
does Docker really allow copying files from _any_ state?

Reply:
see https://github.com/testcontainers/testcontainers-java/issues/3264

Reply:
then please add a switch that returns `true` only on these states. Alternatively, the check can be relaxed to `getContainerId() != null` and then the Docker API call would fail.

Reply:
> bsideup commented on 2 Dec 2020
> @michael-simons I am wondering what will be the behaviour if we remove the check on our side and let the Docker  API call fail instead, so that it is up to Docker to decide whether it is okay to perform the operation or not

Reply:
yes, this.

Reply:
you want a switch statement, or you want to leave it up to docker to return errors? I don't see how these ideas are the same?